crates/graphql/src/schema/query_root.rs

```diff
@@ -537,6 +542,51 @@ impl QueryRoot {
             .collect::<Vec<Wallet>>())
     }
 
+    #[graphql(
+        description = "Get multiple marketplaces; results will be in alphabetical order by subdomain"
+    )]
+    fn marketplaces(
+        &self,
+        context: &AppContext,
+        #[graphql(
+            description = "Return these marketplaces; results will be in alphabetical order by subdomain."
+        )]
+        subdomains: Option<Vec<String>>,
+        #[graphql(description = "Limit for query")] limit: Option<i32>,
+        #[graphql(description = "Offset for query")] offset: Option<i32>,
+    ) -> FieldResult<Vec<Marketplace>> {
+        if subdomains.is_none() && limit.is_none() {
+            return Err(FieldError::new(
+                "You must supply limit, subdomains, or both.",
+                graphql_value!({ "Filters": "subdomains: Vec<String>, limit: i32" }),
+            ));
+        }
+
+        let conn = context.shared.db.get()?;
+        let mut query = store_config_jsons::table
+            .select(store_config_jsons::all_columns)
+            .filter(
+                store_config_jsons::store_address
+                    .ne(all(&context.shared.marketplaces_store_address_exclusions)),
+            )
+            .order(store_config_jsons::name.asc())
+            .into_boxed();
+
+        if let Some(subdomains) = subdomains {
+            query = query.filter(store_config_jsons::subdomain.eq(any(subdomains)));
```

Comment:
If `offset` is not used for this branch it should be an error to have `subdomains.is_some() && offset.is_some()`

Reply:
Will update to error if there's a weird combination of the 3.